### ISSUE TITLE
undefined is not an object (evaluating 'RNGestureHandlerModule.State')

### ISSUE THREAD

**Issue by Albert0405** [kmagiera/react-native-gesture-handler#205]
What i did 
1. npm install --save react-native-gesture-handler
2.  react-native link react-native-gesture-handler
3. Edit my MainActivity
4. Edit my MainApplication
5. import { TapGestureHandler } from 'react-native-gesture-handler';
6. Got the error below

![34484119_1694071200628127_4496503740102082560_n](https://user-images.githubusercontent.com/38030623/40913265-9e74c068-6826-11e8-8bc5-316098311ed3.png)

**Comment by Albert0405** [comment]
```
"dependencies": {
    "@expo/vector-icons": "^6.3.1",
    "axios": "^0.18.0",
    "firebase": "^5.0.3",
    "react": "16.3.1",
    "react-native": "~0.55.2",
    "react-native-gesture-handler": "^1.0.4",
    "react-native-nested-scroll-view": "^6.0.1",
    "react-native-progress": "^3.4.0",
    "react-native-responsive-dimensions": "^1.0.2",
    "react-native-swiper": "^1.5.13",
    "react-native-text": "0.0.7",
    "react-native-vector-icons": "^4.6.0",
    "react-navigation": "^2.0.1",
    "watchman": "^1.0.0"
  }
```

**Comment by victorporof** [comment]
I had a similar issue. I'm using CocoaPods instead of react-native link.

Add this to your fodfile: 
```
pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler/ios'
```

**Comment by Albert0405** [comment]
Hi i solved it now what i did was is to create a new RN project using `react-native init nameOfYourProject` and install the following:

```
npm install --save react-native-gesture-handler
react-native link react-native-gesture-handler
```

and edit my MainActivity and MainApplication:

//MainActivity.java
```
package com.mlguide; // make sure you change it to your project's name, mine is com.mlguide 

import com.facebook.react.ReactActivity;
import com.facebook.react.ReactActivityDelegate;
import com.facebook.react.ReactRootView;
import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;

public class MainActivity extends ReactActivity {

    /**
     * Returns the name of the main component registered from JavaScript.
     * This is used to schedule rendering of the component.
     */
    @Override
    protected String getMainComponentName() {
        return "mlguide"; // this one too
    }

    @Override
    protected ReactActivityDelegate createReactActivityDelegate() {
      return new ReactActivityDelegate(this, getMainComponentName()) {
        @Override
        protected ReactRootView createRootView() {
            return new RNGestureHandlerEnabledRootView(MainActivity.this);
      }
    };
  }
}

```


//MainApplication.java
```
package com.mlguide; // change it

import android.app.Application;

import com.facebook.react.ReactApplication;
import com.oblador.vectoricons.VectorIconsPackage; // Dont mind this. This is another plugin i used in my project
import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.ReactPackage;
import com.facebook.react.shell.MainReactPackage;
import com.facebook.soloader.SoLoader;

import java.util.Arrays;
import java.util.List;

public class MainApplication extends Application implements ReactApplication {

  private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
    @Override
    public boolean getUseDeveloperSupport() {
      return BuildConfig.DEBUG;
    }

    @Override
    protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
          new MainReactPackage(),
            new VectorIconsPackage(), // This one too
            new RNGestureHandlerPackage()
      );
    }

    @Override
    protected String getJSMainModuleName() {
      return "index";
    }
  };

  @Override
  public ReactNativeHost getReactNativeHost() {
    return mReactNativeHost;
  }

  @Override
  public void onCreate() {
    super.onCreate();
    SoLoader.init(this, /* native exopackage */ false);
  }
}

```

**Comment by kmagiera** [comment]
Thanks for reporting @Albert0405 it looks like you've missed some setup steps (namely link command). Since you managed to resolve it I'm going to close this issue. If you think k there is something we can do to make setup instructions cleaner feel free to start a new issue

**Comment by victorporof** [comment]
@kmagiera Lots of people integrate React Native into an existing project instead of starting from scratch. This often means using CocoaPods instead of react-native link. Some instructions for this use case would probably help immensely, especially since the Podfile isn't in the root directory.

**Comment by kmagiera** [comment]
Thanks @victorporof, I well understand the importance of supporting people who rely kn cocoapods for pulling in their dependencies. I just thought react-native link works with cocoapods (I think since RN 52) so didn't see it to be necessary to add that to instructions.

There is a PR open that adds podfile to the root folder. Would that help if we put it there under a "standardized" name?

**Comment by Albert0405** [comment]
I would also like to suggest to update your documentation (Installation Guide). Make it beginner friendly. I was confused on `Android` part where you need to change some codes in it. It says: 
```
Update your main activity (or wherever you create an instance of ReactActivityDelegate), 
so that it overrides the method responsible for creating ReactRootView instance. Then 
use a root view wrapper provided by this library:

import com.facebook.react.ReactActivityDelegate;
import com.facebook.react.ReactRootView;
import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;

public class MainActivity extends ReactActivity {

  // Add the following method to your main activity class
  @Override
  protected ReactActivityDelegate createReactActivityDelegate() {
    return new ReactActivityDelegate(this, getMainComponentName()) {
      @Override
      protected ReactRootView createRootView() {
        return new RNGestureHandlerEnabledRootView(MainActivity.this);
      }
    };
  }
}
```

I was like, What? Where can i find my `main activity` class? Im clueless so i googled it but it would be a time saver if you include the location of the file just like what i did on my solution above

**Comment by kmagiera** [comment]
That's a great suggestions @Albert0405  would you be interested in updating the docs and sending a PR with that? If so I'd be more than happy to merge it

**Comment by pcooney10** [comment]
Hey @kmagiera this was also a little unclear to me so I took a go at updating the docs

**Comment by anchetaWern** [comment]
I'm getting the same issue on iOS:

![screen shot 2018-06-26 at 5 01 49 pm](https://user-images.githubusercontent.com/743603/41901100-9b49a3e0-791f-11e8-8770-b06daad574e1.png)

This is after doing `react-native-link`. So I decided to use cocoapods instead:

```
cd ios
pod init
```

Then I added the following to my `podfile`:

```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'RNRealworldAnimations' do
 
  rn_path = '../node_modules/react-native'
  pod 'yoga', path: "#{rn_path}/ReactCommon/yoga/yoga.podspec"
  pod 'React', path: rn_path, subspecs: [
    'Core',
    'CxxBridge',
    'DevSupport',
    'RCTActionSheet',
    'RCTAnimation',
    'RCTGeolocation',
    'RCTImage',
    'RCTLinkingIOS',
    'RCTNetwork',
    'RCTSettings',
    'RCTText',
    'RCTVibration',
    'RCTWebSocket',
  ]

  pod 'DoubleConversion', :podspec => "#{rn_path}/third-party-podspecs/DoubleConversion.podspec"
  pod 'glog', :podspec => "#{rn_path}/third-party-podspecs/glog.podspec"
  pod 'Folly', :podspec => "#{rn_path}/third-party-podspecs/Folly.podspec"

  pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler/ios'
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == "React"
      target.remove_from_project
    end
  end
end
```

Ran the project:

```
cd ..
react-native run-ios
```

And I got the error: 

```
`PickerIOS` does not exist in the Haste module map
```

Which was fixed by:

```
yarn start --reset-cache
```

But after that, I got the original error back. 

Any ideas?

**Comment by kmagiera** [comment]
Sorry to hear it's happening to you too @anchetaWern 

This problem has clearly to do with the library setup process and indicates that the native code is not being properly included and run in your app.

Unfortunately with the following details it is difficult for me to tell what's going wrong in your case. It seems like you should've run "pod install" or sth after adding stuff to Podfile (not sure if you did but it's not mentioned in your message).

The only advice that I have is to also check in other libraries as this problem is not specific to this one but rather to linking RN library with native code into your project in general. You could try adding some other library (such as react-native-permissions) and checking if it works. If you are familiar with xcode projects I'd suggest to look into the project config and check if gesture handler lib is properly linked

**Comment by ikzjfr0** [comment]
I got the similar problem on iOS, i tried both way (link and cocoapod)

![image](https://user-images.githubusercontent.com/6456056/42123380-9a89697c-7c94-11e8-818d-9eb7685d965e.png)

**Comment by anchetaWern** [comment]
@kmagiera I wasn't really sure how to check if a module is linked in Xcode, so what I did was to just start a new React Native project and copied the existing code over.
I skipped `react-native link` and just proceeded to using the `Podfile` that I used above.

@ikzjfr0 maybe that will solve your issue too. But this method requires you to manually link on Android.

**Comment by ikzjfr0** [comment]
@anchetaWern  starting a new RN project does resolve the problem. However, my project is too big to easily just copying :)

**Comment by gao520sun** [comment]
Unhandled JS Exception: undefined is not an object (evaluating 'RNGestureHandlerModule.State')

**Comment by casper123** [comment]
I was also getting the same issue. What I did was delete the app from my iPhone simulator and then go to Hardware -> Erase all content & settings. After that I reinstalled the app and everything worked fine after that :)

**Comment by nshoes** [comment]
@casper123 Wow! Haha that worked for me, thanks. I have a relatively small RN project, followed the first code example [on the react-navigation website](https://reactnavigation.org/docs/en/hello-react-navigation.html), and hit this error. `react-native link` did not help.

**Comment by achmadk** [comment]
[Manual linking with Xcode](https://facebook.github.io/react-native/docs/linking-libraries-ios) work for me. You can drag `RNGestureHandler.xcodeproj` into Libraries, then drag `libRNGestureHandler.a` into `Build Phases > Link Binary with Libaries`

**Comment by harish-aka-shivi** [comment]
I am following [these](https://facebook.github.io/react-native/docs/integration-with-existing-apps) instructions to integrate react-native to my existing app.

**Comment by harish-aka-shivi** [comment]
I solved it by adding `RNGestureHandlerPackage()` to the `ReactInstanceManagerBuilder`'s list of packages, manually. @kmagiera Is this something that can be added in the documentation, particularly, for the cases when developer is adding react-native to the existing native android apps? I am happy to help.

**Comment by AntonioErdeljac** [comment]
My solution was to wipe out emulator data or try react-native run-android on a completely new emulator. I installed and linked react-native-gesture-handler while my emulator was active and I thought re-running run-android would reset all cache but the emulator app needs to be removed and data wiped out and then try again. Hope this helps someone.

**Comment by yangkezun** [comment]
> My solution was to wipe out emulator data or try react-native run-android on a completely new emulator. I installed and linked react-native-gesture-handler while my emulator was active and I thought re-running run-android would reset all cache but the emulator app needs to be removed and data wiped out and then try again. Hope this helps someone.

thanks

**Comment by tandemloop** [comment]
Did react-native link, wiped data, cleared cache, updated packages, no go. Just using the copied example in starting react-native docs.
What fixed it was @achmadk suggested. The file already existed in Libraries but it didn't exist in Build Phases...

**Comment by MIyulong** [comment]
> I had a similar issue. I'm using CocoaPods instead of react-native link.
> 
> Add this to your podfile:
> 
> ```
> pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler/ios'
> ```

as for me, it doesn't work, i get this `[!] No podspec found for `RNGestureHandler` in `../node_modules/react-native-gesture-handler/ios/``

**Comment by achmadk** [comment]
Change pod path for `RNGestureHandler` to `../node_modules/react-native-gesture-handler`, @MIyulong 

```rb
pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'
```

**Comment by acro5piano** [comment]
I had the same problem in iOS, solved by manual installation:

- `open node_modules/react-native-gesture-handler/ios/`
- finder will show up
- drag the dir `RNGestureHandler.xcodeproj` to `Libraries` in the top of Xcode
- select `only reference` and `create group` if the dialog shows up

I think it is helpful if `Getting Started` > `Installation` section in the document.

![image 2018-12-26 13-11-05](https://user-images.githubusercontent.com/10719495/50431701-1cdec900-0910-11e9-81bf-fe2a38e032bc.png)

**Comment by hichamnaimi** [comment]
If you are using react-link to link your dependencies:
1/ open your ios Podfile and delete all yout linked dependencies: pod 'xxxxxxx', :path => '../node_modules/xxxxx'
2/ Close Xcode
3/ In your /ios folder run "pod update"
4/ In your project source run "react-native link"
5/ Open Xcode and Clean Build Folder from Xcode Menu -> Product
6/ Run your application from Xcode
7/ Link manually the dependency "react-native-gesture-handler" into your Xcode Project following steps in documentation: https://facebook.github.io/react-native/docs/linking-libraries-ios
8/ Now run your application from Xcode, you should be fine.

**Comment by cmcaboy** [comment]
I was able to fix this by following @acro5piano's recommendation, but I had to also add libRNGestureHandler.a to General -> Linked Frameworks and Libraries section.

More or less, I had to manually link. react-native link did not work for my project.

**Comment by Akashsystematix** [comment]
Manual Mehod Will fix the issue.

**Comment by Amos729** [comment]
> I had a similar issue. I'm using CocoaPods instead of react-native link.
> 
> Add this to your podfile:
> 
> ```
> pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler/ios'
> ```

it works for me, thx

**Comment by joydeep33** [comment]
> I had a similar issue. I'm using CocoaPods instead of react-native link.
> 
> Add this to your podfile:
> 
> ```
> pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler/ios'
> ```

What is pod file

**Comment by vinaygode** [comment]
1.  (As first stated by @achmadk)  Add to Podfile: `pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler`
2. It helped me to remove pods and reinstall: `rm -rf Pods; pod install`

**Comment by andresmtz98** [comment]
Nothing from these workarounds work for me :/

**Comment by Ovi** [comment]
I was having same problem in Android Emulator
Here's what I did

- Deleted `node_modules`
- `npm install` (or `yarn install` if you're using yarn)
- `npm install --save react-navigation` (or for yarn `yarn add react-navigation`)
- `npm install --save react-native-gesture-handler` (yarn: `yarn add react-native-gesture-handler`)

- Open YourApp > android > app > build.gradle
into `dependencies` block add the following line 
`implementation project(':react-native-gesture-handler')`

- Open YourApp > android > settings.gradle
add following lines
`
include ':react-native-gesture-handler'
project(':react-native-gesture-handler').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-gesture-handler/android')
`

**Comment by mdehghani65** [comment]
> If you are using react-link to link your dependencies:
> 1/ open your ios Podfile and delete all yout linked dependencies: pod 'xxxxxxx', :path => '../node_modules/xxxxx'
> 2/ Close Xcode
> 3/ In your /ios folder run "pod update"
> 4/ In your project source run "react-native link"
> 5/ Open Xcode and Clean Build Folder from Xcode Menu -> Product
> 6/ Run your application from Xcode
> 7/ Link manually the dependency "react-native-gesture-handler" into your Xcode Project following steps in documentation: https://facebook.github.io/react-native/docs/linking-libraries-ios
> 8/ Now run your application from Xcode, you should be fine.

after hours, only this worked for me

**Comment by axdamx** [comment]
@AmirhosseinBayat tricks does solved the problem! Thanks bro!

**Comment by FadiAboMsalam** [comment]
@achmadk finally after spending many hours linking manually solved the issue for me thanks a million

**Comment by Dhirajbhujbal** [comment]
Manual Integration of the Navigation project resolved this issue.
2 options we can use pods or we can do the manual integration.

PS: This will occur only v3.0

**Comment by ggoldens** [comment]
I've sent many hours on this. Using pods worked to me.

**Comment by AGPatel** [comment]
@Albert0405 
I change all my file MainActivity.java and and MainApplication.java as you give above code but still error is same like   " **null is not an object (evaluating 'RNGestureHandlerModule.state')**"  

**package.json**
`{
  "name": "ak1",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest"
  },
  "dependencies": {
    "react": "16.6.3",
    "react-native": "0.58.5",
    "react-native-gesture-handler": "^1.0.16",
    "react-native-vector-icons": "^6.3.0",
    "react-navigation": "^3.3.0"
  },
  "devDependencies": {
    "babel-core": "7.0.0-bridge.0",
    "babel-jest": "24.1.0",
    "jest": "24.1.0",
    "metro-react-native-babel-preset": "0.52.0",
    "react-test-renderer": "16.6.3"
  },
  "jest": {
    "preset": "react-native"
  }
}`

**MainActivity.java**
`package com.ak1;

import com.facebook.react.ReactActivity;
import com.facebook.react.ReactActivityDelegate;
import com.facebook.react.ReactRootView;
import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;

public class MainActivity extends ReactActivity {

    /**
     * Returns the name of the main component registered from JavaScript.
     * This is used to schedule rendering of the component.
     */
    @Override
    protected String getMainComponentName() {
        return "ak1";
    }

    @Override
    protected ReactActivityDelegate createReactActivityDelegate() {
        return new ReactActivityDelegate(this, getMainComponentName()) {
            @Override
            protected ReactRootView createRootView() {
                return new RNGestureHandlerEnabledRootView(MainActivity.this);
            }
        };
    }
}`

**MainApplication.java**
`package com.ak1;

import android.app.Application;

import com.facebook.react.ReactApplication;
import com.oblador.vectoricons.VectorIconsPackage;
import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.ReactPackage;
import com.facebook.react.shell.MainReactPackage;
import com.facebook.soloader.SoLoader;

import java.util.Arrays;
import java.util.List;

public class MainApplication extends Application implements ReactApplication {

  private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
    @Override
    public boolean getUseDeveloperSupport() {
      return BuildConfig.DEBUG;
    }

    @Override
    protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
          new MainReactPackage(),
            new VectorIconsPackage(),
            new RNGestureHandlerPackage()
      );
    }

    @Override
    protected String getJSMainModuleName() {
      return "index";
    }
  };

  @Override
  public ReactNativeHost getReactNativeHost() {
    return mReactNativeHost;
  }

  @Override
  public void onCreate() {
    super.onCreate();
    SoLoader.init(this, /* native exopackage */ false);
  }
}
`

and this is my  **App.js** file
`
import React, {Component} from 'react';
import {Platform, Button, StyleSheet, Text, View} from 'react-native';
import { Ionicons } from "react-native-vector-icons/Ionicons";
import { createStackNavigator, createBottomTabNavigator, createAppContainer } from 'react-navigation';
import { TapGestureHandler } from 'react-native-gesture-handler';

class HomeScreen extends React.Component {
  static navigationOptions = {
    title: 'Welcome to home',
  };
  render() {
    return (
      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
        <Text>Home!</Text>
        <Button
          title="Go to Settings"
          onPress={() => this.props.navigation.navigate('Settings')}
        />
        <Button
          title="Go to Details"
          onPress={() => this.props.navigation.navigate('Details')}
        />
      </View>
    );
  }
}

class SettingsScreen extends React.Component {
  static navigationOptions = {
    title: 'Welcome to SettingsScreen',
  };
  render() {
    return (
      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
        <Text>Settings!</Text>
        <Button
          title="Go to Home"
          onPress={() => this.props.navigation.navigate('Home')}
        />
        <Button
          title="Go to Details"
          onPress={() => this.props.navigation.navigate('Details')}
        />
      </View>
    );
  }
}

class DetailsScreen extends React.Component {
  render() {
    return (
      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
        <Text>Details!</Text>
      </View>
    );
  }
}
const HomeStack = createStackNavigator({
  Home: { screen: HomeScreen },
  Details: { screen: DetailsScreen },
});

const SettingsStack = createStackNavigator({
  Settings: { screen: SettingsScreen },
  Details: { screen: DetailsScreen },
});

const App =  createAppContainer(createBottomTabNavigator(
  {
    Home: { screen: HomeStack },
    Settings: { screen: SettingsStack },
  },
  {
    defaultNavigationOptions: ({ navigation }) => ({
      tabBarIcon: ({ focused, tintColor }) => {
        const { routeName } = navigation.state;
        let iconName;
        if (routeName === 'Home') {
          iconName = `ios-information-circle${focused ? '' : '-outline'}`;
        } else if (routeName === 'Settings') {
          iconName = `ios-options${focused ? '' : '-outline'}`;
        }
        return <Ionicons name={iconName} size={25} color={tintColor} />;
      },
    }),
    tabBarOptions: {
      activeTintColor: 'tomato',
      inactiveTintColor: 'gray',
    },
  }
));
export default App ;
`
so,how can i remove error

**Comment by AGPatel** [comment]
Yes i got it ,
you need to change the **setting.gradle** from android

from->
```
include ':react-native-gesture-handler'
project(':react-native-gesture-handler').projectDir = new File(rootProject.projectDir, '..\node_modules\react-native-gesture-handler\android')
```
to ->
```
include ':react-native-gesture-handler'
project(':react-native-gesture-handler').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-gesture-handler/android')
```

if you install anything from **npm install...** so first check the  **setting.gradle** and it's same like **backward slash(\)** so change to **forward slash (/)**

**Comment by Nizzam123** [comment]
I have tried all the solution given above. But I still failed and getting this error.

This is my env :
"react": **"16.6.3"**,
"react-native": **"0.58.5"**,
"react-native-gesture-handler": **"^1.0.16"**,
"react-navigation": **"^3.3.2"**

npm : **v6.8.0**

![screenshot 2019-02-26 at 5 41 00 pm](https://user-images.githubusercontent.com/37056592/53402990-bc423a00-39ed-11e9-8913-66043307b41d.png)

Appreciate any help. Thanks.

**Comment by Dhirajbhujbal** [comment]
@Nizzam123  try to manual integration of the navigation lib.

**Comment by Nizzam123** [comment]
@Dhirajbhujbal I did it, plus also reinit the native project and install the library, the result still same. Sigh.

**Comment by Dhirajbhujbal** [comment]
follow the below step

install react-native-navigation lib 3.0 
1 npm install --save react-navigation
2 npm install --save react-native-gesture-handler
3 react-native link 

check whether all configuration is fine or not if still getting the same issue then you might be missing so the configuration so after step 2 try to add RNGestureHandler.xcodeproj to your lib. a folder in the Xcode then add libRNGestureHandler.a file into 'Link Binary With libraries' also in the build setting put header Search path as '$(SRCROOT)/../node_modules/react-native-navigation/lib/ios'.

and clean the Xcode and try to build again hope this will work

Thanks,
Dhiraj

**Comment by RicardoBer** [comment]
In iOS

Add this to your Podfile

`pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'`

**Comment by Nizzam123** [comment]
@RicardoBer Yes, I add that. The error still appear.

**Comment by genogeno** [comment]
@Nizzam123 
After you add the code to Podfile and executed pod install.
Try running your metro bundler
`
npm run start
`
Can you check your metro bundler logs.  Do you see anything related with 

`
Error: jest-haste-map: Haste module naming collision:
  Duplicate module name: react-native
`

**Comment by mangndi** [comment]
refering to react navigation (https://reactnavigation.org/docs/en/getting-started.html), some step should be done, as follow 
1. npm install --save react-navigation
2. npm install --save react-native-gesture-handler
3. react-native link react-native-gesture-handler

then re-run again

**Comment by anubhav-r-mathur** [comment]
This will solve the problem make sure you run npm install after adding the lib in react project 
pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'

**Comment by ElliDy** [comment]
I still have this problem with react-native 0.75.4 and react-native-gesture-handler 2.20.0. Can somebody help me with it?
